pyk/src/pyk/kcfg.py

```diff
@@ -347,6 +356,25 @@ def edges(self, *, source_id: Optional[str] = None, target_id: Optional[str] = N
 
         return [edge for edge in res if not target_id or target_id == edge.target.id]
 
+    def contains_edge(self, edge: Edge) -> bool:
+        if other := self.edge(source_id=edge.source.id, target_id=edge.target.id):
+            return edge == other
+        return False
+
+    def create_edge(self, source_id: str, target_id: str, condition: KInner = TRUE, depth=1) -> Edge:
```

Comment:
I'm not sure it makes sense to have a default `depth` for this one actually. Maybe we should force the user to supply the `depth`?

Reply:
Done. (I also removed the default value for `condition`, otherwise I would have had to change the order of arguments.)